### PR TITLE
Blogpost mark tutorials

### PR DESCRIPTION
**Pull request by DennisFriedl**

### CODE REVIEW

File: _posts/2024-07-17-Beethoven-Tutorials.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: "New Tutorials released for the Beethoven Annotator App"
+authors: "Mark Saccomano"
+excerpt: "Try out the new step-by-step guide to creating digital annotations on digital resources using MEI and Linked Data."
+teaserImage: "/assets/blog/2024-07-17/teaser.jpg"
+---
+
+The online Annotator App developed for the ViFE project [Beethoven in the House](https://domestic-beethoven.eu/) now has [documentation and tutorials available](https://domestic-beethoven.eu/tutorials/), thanks to funding obtained from NFDI4Culture. Beethoven in the House was a three-year collaborative research project between Edirom, the Oxford e-Research Centre, and the Beethoven Archive Research Center in Bonn. The project website now features tutorials that illustrate the use of the app, broken down into 5 short steps.
```
Comment (peterstadler) [inline _posts/2024-07-17-Beethoven-Tutorials.md:9]: I'd suggest to replace "Edirom" with "Paderborn University" here:
```suggestion
The online Annotator App developed for the ViFE project [Beethoven in the House](https://domestic-beethoven.eu/) now has [documentation and tutorials available](https://domestic-beethoven.eu/tutorials/), thanks to funding obtained from NFDI4Culture. Beethoven in the House was a three-year collaborative research project between Paderborn University, the Oxford e-Research Centre, and the Beethoven Archive Research Center in Bonn. The project website now features tutorials that illustrate the use of the app, broken down into 5 short steps. 
```

File: _posts/2024-07-17-Beethoven-Tutorials.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+The project’s core development question was how to create an app that could support annotation of the same passage of music as it appears in different arrangements. The data model created for this task was therefore multilayered, so that musical structures could be captured at varying degrees of scale and at varying levels of abstraction. When it came time to develop a user interface for the prototype app, a certain level of complexity was unavoidable. The researchers wanted to easily confirm that all components were working together as intended. The user interface, therefore, is perhaps not quite as intuitive as one might expect from an app ready for public use. Hence the need for tutorials.
+
+The tutorials are multi-model: They include text, screen captures, graphics, and screencasts. They are in beta version at the moment and posted on the Beethoven in the House website.
```
Comment (peterstadler) [inline _posts/2024-07-17-Beethoven-Tutorials.md:13]: Shouldn't that read "multi-modal"?
```suggestion
The tutorials are multi-modal: They include text, screen captures, graphics, and screencasts. They are in beta version at the moment and posted on the Beethoven in the House website. 
```

**Review by peterstadler — COMMENTED** [review]
Thanks for the blog post! 
I only left some minor suggestions, please have a look @mss2221

File: _posts/2024-07-17-Beethoven-Tutorials.md (hunk 1 above)
Comment (mss2221) [reply _posts/2024-07-17-Beethoven-Tutorials.md:9]: Looks good. I meant to ask about that...

File: _posts/2024-07-17-Beethoven-Tutorials.md (hunk 2 above)
Comment (mss2221) [reply _posts/2024-07-17-Beethoven-Tutorials.md:13]: * face palm emoji *

**Review by peterstadler — APPROVED** [review]

**Pull request merged**